### DISCUSSION TITLE
Unable to use childprocess.exec in ts script

### DISCUSSION THREAD

**Discussion by 0xMySt1c** [oven-sh/bun#2887] [Q&A] [answered]
import util from "util";
import childProcess from "child_process";

const exec = util.promisify(childProcess.exec);

const { stdout } = await exec("which bun");

console.log(stdout) - has no output

**Comment by 0xMySt1c** [comment]
not sure if these modules are not implemented yet or this is a discrepancy in bun.
open to suggestions to work around this.

**Comment by vjpr** [comment, marked as answer]
I think it is promisified by default.

**FYI**

API Status: https://bun.sh/docs/runtime/nodejs-apis#node_child_process

Comparison of Node vs Bun: https://github.com/privatenumber/compare-bun-node#child_process
